src/pages/overview/document-generation-api/stylingformattingtags.md

```diff
@@ -90,16 +92,36 @@ JSON representation of the input data:
 
 - <img\> (image)
 
+- [<p\> (para)](/overview/document-generation-api/stylingformattingtags/#how-to-create-paragraphs-in-document)
+
+
 JSON representation of the input data:
 
 ```json
 {
-  "Notes Section": "Has the Board of directors established a basic policy <span style=\"font-size: 12pt;\">regarding legal compliance <b><i>(hereinafter referred to as the Legal Compliance policy)</i></b><a href=\"http://www.samplewebsite.com/\">Board</a> in accordance to the institution's <br><u>corporate management policy</u> and disseminated it throughout the institution"
+  "Notes Section": "<p>Has the Board of directors established a basic policy</p> <span style=\"font-size: 12pt;\">regarding legal compliance <b><i>(hereinafter referred to as the Legal Compliance policy)</i></b><a href=\"http://www.samplewebsite.com/\">Board</a> in accordance to the institution's <br><u>corporate management policy</u> and disseminated it throughout the institution"
+}
+```
+![Styling Tags Sample with style for font-size attribute added to span tag](../images/styling_tags_sample_ptag.png)
+
+![Styling Tags output when style for font-size attribute was added to span tag](../images/styling_tags2.png)
+ 
+#### How to create paragraphs in document?
+
+New paragraphs can be created using `<p>` in input data
```

Comment:
This needs to be rephrased like, take reference from the explanation of other html tags, use the similar language.

Reply:
Done.